.history/scripts/biomedical/VirtualMetabolicHuman/format_reaction_20210704105952.py

```diff
@@ -0,0 +1,221 @@
+import pandas as pd
```

Comment:
Can we remove all these format_reaction_2021xyz.py files?

Reply:
Done ! Sorry about that, I accidentally pushed the .history files too